addons/website_event/static/src/js/display_timer_widget.js

```diff
@@ -17,19 +17,19 @@ publicWidget.registry.displayTimerWidget = publicWidget.Widget.extend({
     start: function () {
         var self = this;
         return this._super.apply(this, arguments).then(function () {
-            self.options = self.$el.data();
-            self.preCountdownDisplay = self.options["preCountdownDisplay"];
-            self.preCountdownTime = self.options["preCountdownTime"];
+            self.options = self.el.dataset;
+            self.preCountdownDisplay = Boolean(self.options["preCountdownDisplay"]);
```

Comment:
`Boolean("false")` returns `true`, so probably not what you expect. If I'm not wrong `self.preCountdownDisplay` is only used once below, just check if `=== "true"`.